test/integration/kubernetes.spec.ts

```diff
@@ -29,9 +29,10 @@ async function teardown(): Promise<void> {
   console.log('Removed "kind" network');
 }
 
-beforeAll(teardown);
 afterAll(teardown);
 
+test('clean up environment on start', teardown);
```

Comment:
the beforeAll doesnt work?

Reply:
Yes, seems like this setup-after-env only works for test() cases. The problem we had was in the beforeAll() where setup could fail but tests still continued. I tried to throw in the beforeAll() and it still continued doing stuff!